packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -0,0 +1,194 @@
+    const value = settings?.[property] || '';
+    const onChange = (value: string) => {
+      const patched = {
+        ...settings,
+        [property]: Number(value),
+      } as SyncSettings;
+      setSyncSettings(patched);
+    };
+    return (
+      <NumericTextInput
+        value={value}
+        onChange={e => onChange(e.target.value)}
+        disabled={isDisabled}
+      />
+    );
+  };
+
+  return (
+    <form style={{ width: '100%' }}>
+      <Setting
+        title={t('label.settings-url')}
+        component={<StringSyncSetting property="url" />}
+      />
+      <Setting
+        title={t('label.settings-username')}
+        component={<StringSyncSetting property="username" />}
+      />
+      <Setting
+        title={t('label.settings-password')}
+        component={
+          <StringSyncSetting
+            property="password"
+            type="password"
+            autocomplete="sync-password"
+          />
+        }
+      />
+      <Setting
+        title={t('label.settings-interval')}
+        component={<NumericSyncSetting property="intervalSec" />}
+      />
+      <Setting
+        title={t('label.settings-central-site-id')}
+        component={<NumericSyncSetting property="centralServerSiteId" />}
+      />
+      <Setting
+        title={t('label.settings-site-id')}
+        component={<NumericSyncSetting property="siteId" />}
+      />
+      <Grid item justifyContent="flex-end" width="100%" display="flex">
+        <LoadingButton
+          isLoading={isSaving}
+          startIcon={<SaveIcon />}
+          variant="contained"
+          sx={{ fontSize: '12px' }}
+          disabled={!isValid}
+          onClick={onSave}
+        >
+          {t('button.save')}
+        </LoadingButton>
+      </Grid>
+    </form>
+  );
+};
+
+export const SyncSettings = ({}) => {
+  const { data, isLoading, isError } = useHost.utils.settings();
+  const { mutate, isLoading: isSaving } = useHost.sync.update();
+  const { mutateAsync: serverRestart } = useHost.utils.restart();
+  const t = useTranslation('common');
+  const [syncSettings, setSyncSettings] = useState<SyncSettings | null>(null);
+  const { success, info } = useNotification();
+
+  const currentSettings = {
+    centralServerSiteId: data?.syncSettings?.centralServerSiteId || 1,
+    intervalSec: data?.syncSettings?.intervalSec || 10,
+    password: '',
+    siteHardwareId: data?.syncSettings?.siteHardwareId || '',
+    siteId: data?.syncSettings?.siteId || 2,
+    url: data?.syncSettings?.url || '',
+    username: data?.syncSettings?.username || '',
+  };
+  const settings = data?.syncSettings
+    ? { ...currentSettings, ...syncSettings }
+    : undefined;
+
+  const onSave = () => {
+    if (!syncSettings) return;
+    const successSnack = success(t('success.sync-settings'));
+    const restart = async () => {
+      successSnack();
+      await serverRestart(); // returns 'Restarting'
+      const infoSnack = info(t('info.server-restarting'));
```

Comment:
discarding the 'Restarting' message in order to provide a translatable message. Should check for the response but didn't want to be limiting ourselves to a hard coded string at this stage (I'm not confident that the API response won't change at some point!) so went with the slightly looser approach and will 'info' when the request is sent